common/src/ui/RecentDocumentListBox.h

```diff
@@ -46,6 +46,7 @@ private slots:
   QString subtitle(size_t index) const override;
 
   void doubleClicked(size_t index) override;
+  void keyPressEvent(QKeyEvent* event) override;
```

Comment:
"Update <Filename>" isn't a good commit message. Please squash the two commits together and give them a proper description. The subject should explain what the commit changes in a sentence, and the body should give additional details (if necessary). Maybe check other commit messages to get a better feeling for what I mean.